src/reduce.test.js

```diff
@@ -11,9 +11,39 @@ describe('reduce', () => {
     delete Array.prototype.reduce2;
   });
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(reduce).toBeInstanceOf(Function);
+  });
+
+  it('should not call a callback for an empty array', () => {
+    const f = jest.fn();
+
+    expect(f).not.toHaveBeenCalled();
+  });
+
+  it('should call a callback once per item', () => {
+    const f = jest.fn();
+
+    [1, 2, 3].reduce(f, 0);
```

Comment:
Use a custom implementation of 'reduce' method instead of the native method
![image](https://github.com/mate-academy/qa_array-method-reduce/assets/96409563/73f8ed70-e1a4-48a3-9c4c-ee0ea30f7a28)